src/swaps/astroport/AstroportSwapProvider.ts

```diff
@@ -136,11 +136,12 @@ class AstroportSwapProvider extends SwapProvider {
     try {
       const tx = await client.chain.getTransactionByHash(swap.swapTxHash);
       if (tx && tx.confirmations && tx.confirmations > 0) {
-        const { status } = await client.getMethod('getTransactionByHash')(swap.swapTxHash);
+        const { status } = tx;
         this.updateBalances(network, walletId, [swap.from]);
+
         return {
           endTime: Date.now(),
-          status,
+          status: status === TxStatus.Success ? 'SUCCESS' : 'FAILED',
```

Comment:
I think we can remove this check here and just leave it `status`, if status is equal to TxStatus.Success for example that means `status` will equal to `SUCCESS`:

https://github.com/liquality/chainify/blob/8257ba10caa513086c484219027a49756d6e3228/packages/types/lib/Transaction.ts#L40

Reply:
The issue is that there is no concept of `PENDING` that's why we either need success or failed